kornia/onnx/sequential.py

```diff
@@ -0,0 +1,150 @@
+from typing import Optional, Union
+
+from kornia.core.external import numpy as np
+from kornia.core.external import onnx
+from kornia.core.external import onnxruntime as ort
+
+__all__ = ["ONNXSequential"]
+
+
+class ONNXSequential:
```

Comment:
I would call it just `ONNX `, as pytorch user I initially thought that is a wrap of nn.Sequential, which seems it not but with more powerful features

Reply:
This behaves like the nn.Sequential. Or maybe ONNXChainer or ONNXCombiner? those are more boring names it looks.

Reply:
ok, let's start by sequential, but i potentially foresee non sequential pipelines

Reply:
Ah. Yes, but this is more complex to handle. Not as convenient as this one-to-one mapping. another class maybe better for the future.